src/views/ListDetails.tsx

```diff
@@ -156,6 +158,8 @@ export default function ListDetails() {
 		const date = dateRef.current?.value;
 		const comment = commentRef.current?.value || null;
 
+		console.log(date);
```

Comment:
Left from debugging?

Reply:
Yes, fixed now and also found another one 😅